dispatch/dispatch.go

```diff
@@ -81,6 +81,7 @@ type AlertBlock struct {
 type APIAlert struct {
 	*model.Alert
 	Status types.AlertStatus `json:"status"`
+	ID     string            `json:"id"`
```

Comment:
I do not think `id` is the right naming. An `id` would give it the notion of uniqueness and uniqueness over time. An alert can appear/disappear and even alerting rules get changed with or without the labels of the alert changing.

Since we all refer to it as this, why not `fingerprint`?